.github/workflows/coverage.yml

```diff
@@ -21,11 +21,12 @@ jobs:
       - name: 'Install dependencies'
         run: ./ci/linux_ci_setup.sh
       - name: 'Run coverage'
+        continue-on-error: true
         run: |
           export PATH=/usr/lib/llvm-10/bin:$PATH
           export CC=clang
           export CXX=clang++
-          export COVERAGE_THRESHOLD=95
+          export COVERAGE_THRESHOLD=97
```

Comment:
Let's not fold this into this change. I'd also like to discuss before we continue to push this upwards.

Reply:
> Testing: testing by setting the coverage bar too high in an earlier commit.

Using to test that the report does in fact upload when caused by failing to meet the threshold. I'll revert once the job runs.

Reply:
Gotcha - missed the comment above. 👍